lib/src/main/kotlin/com/wire/integrations/jvm/service/WireApplicationManager.kt

```diff
@@ -212,16 +218,21 @@ class WireApplicationManager internal constructor(
      * @return The encryption key used to encrypt the asset. Might be useful if you want to
      * download the file later, other clients will receive the same key automatically.
      */
+    @Suppress("LongParameterList")
     fun uploadAndSendMessage(
```

Comment:
Not strictly part of this PR, I gave these names but please rename to sendAsset and sendAssetSuspending or something more clear to external devs